crates/api_models/src/payments.rs

```diff
@@ -2266,6 +2283,70 @@ pub enum BankTransferData {
     },
 }
 
+#[derive(Eq, PartialEq, Clone, Debug, serde::Deserialize, serde::Serialize, ToSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum RealTimePaymentData {
+    Fps {
+        /// The two-letter ISO country code for SEPA and BACS
+        #[schema(value_type = CountryAlpha2, example = "US")]
+        country: Option<api_enums::CountryAlpha2>,
```

Comment:
This is not required as we are already collecting payment_method_data.billing details

Reply:
Removed country from all real time payment methods.